packages/mobile/src/components/audio/Audio.tsx

```diff
@@ -703,6 +711,10 @@ export const Audio = () => {
     }
   }, [playbackState, playing])
 
+  const handleStop = useCallback(async () => {
+    await TrackPlayer.reset()
```

Comment:
If we aren't doing anything with the result, I don't think it needs to be `async`? I'm fairly certain that from React's standpoint this isn't going to block anything or cause the UI to wait until the callback is finished.

Reply:
nice good call